README.md

```diff
@@ -16,6 +16,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [ReactFoo-Delhi](https://reactfoo.in/2018-delhi/) | 18 Aug | TBA | React, alternatives to React, React Native and front-end engineering. |
 | [DevConf.in 2018](https://devconf.info/in) | 4-5 Aug | Christ University - Bengaluru | Annual Developers' Conference organised by Red Hat for local FOSS community |
 | [Deccan RubyConf 2018](https://www.deccanrubyconf.org/) | 4 Aug | Sheraton Grand Hotel, Pune | Pune extension of RubyConf India organized by same folks organizing RubyConf India |
+| [Container Conference 2018](https://www.containerconf.in/) | 3-4 Aug | Radisson Blu (Park Plaza), ORR, Bangalore | Container Conference is a developer focused conference on Docker, Managed Kubernetes/Container Services, Service Mesh, Kubernetes |
```

Comment:
can you edit this to just `Container Conference` instead of `Container Conference 2018`. lgtm otherwise :+1:

Reply:
@nikhita 

Done,
And thanks a lot for maintaining this list